Feat: Webinars & Events Page (#80)
EOPF-Explorer/landing-page: webinars-events/webinar-2-advanced-analysis.md

---

> title: EOPF Explorer Webinar #2 - Advanced Analysis with Sentinel Zarr

Reviewer: ```suggestion
title: "EOPF Explorer Webinar #2 - Advanced Analysis with Sentinel Zarr"
```

layout: page
---

# EOPF Explorer Webinar #2 - Advanced Analysis with Sentinel Zarr

<div class="padding round margin-bottom row">
    <div class="max">
      <div>April 20, 2025 | 15:00-16:00 CET</div>
    </div>
    <a href="https://buytickets.at/thrivegeo/2073515/r/explorerweb" target="_blank" class="button primary">Register Now</a>
</div>

> Following the launch session focused on discovery and screening, this second webinar dives deeper into analysis workflows enabled by the EOPF Sentinel Zarr Explorer.

Reviewer: ```suggestion
While the first webinar focused on discovery and screening, this second one dives deeper into analysis workflows enabled by the EOPF Sentinel Zarr Explorer.
```

If Webinar #1 showed you what is possible, Webinar #2 explores how to go further.

> 

Reviewer: ```suggestion
```

> ### What This Session Focuses On

Reviewer: ```suggestion
### What you'll see in this webinar
```

This session will demonstrate how the EOPF Explorer supports the full user journey, from initial discovery to actionable analysis, without requiring traditional download-heavy workflows.

### Why This Matters

Zarr is the planned future distribution format for Copernicus Sentinel data.

The EOPF Explorer demonstrates how enhanced GeoZarr conventions unlock:
- Faster analysis directly in the browser
- Scalable cloud-native processing
- Reduced friction between exploration and insight
- Open standards-based interoperability

For developers, this means access to:
- Client-side rendering (OpenLayers + WebGL)
- Server-side visualization (TiTiler)
- Open specifications (STAC, GeoZarr conventions)

For users and researchers, this means:
- Immediate analytical capability
- Less overhead
- Faster experimentation and iteration

### Addressing Common Questions

- How the Explorer relates to the Zarr Sample Service
- What is evolving in the GeoZarr conventions
- When broader Sentinel distribution in this enhanced format may happen
- How you can engage now and shape the direction of EOPF

This is an exciting phase of development: community input matters.

> 

Reviewer: We won't address these in detail in the presentation, I think.

```suggestion
```
